File: dto/TrustHistoryItem.java
package com.servitrust.dto;

public class TrustHistoryItem {
    private Long id;
    private Long providerId;
    private Double delta;
    private String reason;
    private Double scoreAfter;
    private String createdAt;

    public TrustHistoryItem(Long id, Long providerId, Double delta, String reason, Double scoreAfter, String createdAt) {
        this.id = id;
        this.providerId = providerId;
        this.delta = delta;
        this.reason = reason;
        this.scoreAfter = scoreAfter;
        this.createdAt = createdAt;
    }

    public Long getId() { return id; }
    public Long getProviderId() { return providerId; }
    public Double getDelta() { return delta; }
    public String getReason() { return reason; }
    public Double getScoreAfter() { return scoreAfter; }
    public String getCreatedAt() { return createdAt; }
}

File: model/TrustHistoryItem.kt
package com.servitrust.app.model

import com.google.gson.annotations.SerializedName

data class TrustHistoryItem(
    @SerializedName("id")
    val id: Long,

    @SerializedName("providerId")
    val providerId: Long,

    @SerializedName("delta")
    val delta: Double,

    @SerializedName("reason")
    val reason: String,

    @SerializedName("scoreAfter")
    val scoreAfter: Double,

    @SerializedName("createdAt")
    val createdAt: String
)